SEED-platform/seed
# refactor(better): add new saved fields

#### Any background context you want to provide?
SEED is improving integration with BETTER

#### What's this PR do?
- stores additional fields into the PropertyState from the BETTER analysis:
  - store each measure recommendation
  - store GHG reduction (note it's in kgCO2e not MTCO2e b/c that's what they give us)
  - min model R**2 (indicator of model quality)
  - SEED analysis ID (so users can go to that analysis to view more stuff about it)

#### How should this be manually tested?
Upload the file below, map all required fields. Import the meter data too. Select one or more properties and run the better analysis ([make sure your org settings has the token](https://github.com/SEED-platform/seed/pull/2788)).

Edit your inventory list columns settings to view the BETTER columns, these columns should be properly populated after running the analysis.

[PM_BETTER_PORTFOLIO_INPUT.xlsx](https://github.com/SEED-platform/seed/files/6932774/PM_BETTER_PORTFOLIO_INPUT.xlsx)


#### What are the relevant tickets?
#2778 

#### Screenshots (if appropriate)
New columns
<img width="1439" alt="Screen Shot 2021-08-04 at 12 02 25 PM" src="https://user-images.githubusercontent.com/18518728/128216362-154aea6e-f4c1-4678-a609-e84ecb54f7c3.png">

Inventory list with new columns
<img width="1440" alt="Screen Shot 2021-08-04 at 12 03 46 PM" src="https://user-images.githubusercontent.com/18518728/128216396-acbec38d-d356-45a7-b3b0-f38eef710a5e.png">
